benchmark/profile_restful_api.py

```diff
@@ -154,9 +154,8 @@ def process_request(self,
         stats = []
         while not res_queue.empty():
             session_id, _stats = res_queue.get()
-            # print(f'\n{"-" * 50}\n'
-            #       f'session {session_id} stats: \n{_stats}\n{"-" * 50}\n')
-            stats.append(np.array(_stats))
+            if len(_stats) != 0:
+                stats.append(np.array(_stats))
```

Comment:
when could this happen? Prompts less than concurrency?

Reply:
> when could this happen? Prompts less than concurrency?

Actually, this is something I encountered during the development of Medusa. I don't remember the specific reproduction conditions very well. At that time, when I saw that other benchmarks had this check, adding it fixed the issue.